Pages/registration/Registration.razor

```diff
@@ -33,8 +33,8 @@
 					                    <ValidationMessage For="@(() => user.ConfirmPassword)" />
 
 											                    <label class="form-label" for="role">Role</label>
-																                    <InputSelect @bind-Value="user.Role" class="form-control" id="role">
-                        <option value="">--SELECT</option>
+																                    <InputSelect @bind-Value="user.Role" class="form-select" id="role">
```

Comment:
With `class=form-control` the select didn't have the little downwards pointing arrow on the right. Without it, it is not clear upon looking that this is a select.

Reply:
So it is solved then?

Reply:
Suggestion pushed. Tell me if it fits (actually, I don't see the difference, but it has the bootstrap property, maybe it affects it somehow).

Reply:
More about this: https://stackoverflow.com/questions/39056651/select-control-is-missing-its-arrow-icon-on-a-form-using-bootstrap-with-firefox